index.js

```diff
@@ -306,10 +306,14 @@ async function wasm_pack(cx, state, dir, source, id, options) {
                     ${prelude}
 
                     export default async (opt = {}) => {
-                        let {importHook} = opt;
+                        let {importHook, serverPath} = opt;
 
                         let path = ${import_wasm};
 
+                        if (serverPath != null) {
+                            path = serverPath + path.split('/').pop();
```

Comment:
```suggestion
                            path = serverPath + /[^\/\\]*$/.exec(path)[0];
```

I think we should handle `\` as well, for extra robustness.